pkg/crypto/hash.go

```diff
@@ -0,0 +1,83 @@
+package crypto
+
+import (
+	"bytes"
+	"encoding/hex"
+	"encoding/json"
+	"hash"
+	"io"
+	"strings"
+
+	"golang.org/x/crypto/sha3"
+	"golang.org/x/sync/errgroup"
+)
+
+var (
+	// DefaultHasher is the default implementation for hashing things
```

Comment:
This doc string should be expanded to say what properties this hasher has

Reply:
:+1: done